common/src/main/java/earth/terrarium/cadmus/common/claims/ClaimSettings.java

```diff
@@ -74,4 +90,28 @@ private static TriState getTriState(CompoundTag tag, String key) {
         if (!tag.contains(key)) return TriState.UNDEFINED;
         return TriState.values()[tag.getByte(key)];
     }
+
+    public TriState canBreak() {return canBreak;}
```

Comment:
These being 1 lines look really weird change them to have return on its own line

Reply:
surprised the formatter doesn't catch that